docs/administration/upgrade/to_v6.0.adoc

```diff
@@ -1,12 +1,33 @@
-= Upgrading to Logging v6.0
+= What's New In Logging 6.0
 
 == Overview
-Logging v6.0 is a major change from earlier releases and is the realization of several longstanding goals of Cluster Logging:
+Logging version 6.0 is a major change from earlier releases and is the realization of several longstanding goals.
 
-* Distinct operators to support logging components (e.g. collectors, storage, visualization)
-* Remove support of managed log storage and visualization based upon the Elastic products (i.e. Elasticsearch, Kibana)
-* Remove support of Fluentd log collector implementation
-* Remove support of the ClusterLogging.logging.openshift.io and ClusterLogForwarder.logging.openshift.io
+The main highlights of these changes are:
+
+* There is now distinct operators to support separate logging components (e.g. collectors, storage, visualization)
+* Cluster Logging Operator no longer implements managed log storage and visualization based upon the Elastic products (i.e. Elasticsearch, Kibana)
+* CLO has removed support of Fluentd log collector implementation
+* API changes remove support of CRD from API Group `logging.openshift.io/v1` of `kind: ClusterLogging` and `kind: ClusterLogForwarder`
```

Comment:
I don't think most admins understand CRD.  This, IMO, would be better worded closer to the original statement:

Introduce the ClusterLogForwarder.observability.openshift.io API to replace both ClusterLogging.logging.openshift.io and ClusterLogForwarder.logging.openshift.io APIs

Though even :up: is slightly misleading since the CL resource represented something completely separate an the new API isn't really replacing it; it sort of replaces a very specific use-case